tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -41,7 +41,25 @@
 <template is="dom-if" if="[[_dataNotFound]]">
   <div style="max-width: 540px; margin: 80px auto 0 auto;">
     <h3>No profile data was found.</h3>
-    <p>The dashboard is internal only; coming soon!</p>
+    <p>To collect a profile, you need to run your model on Google Cloud TPUs and
+    capture the trace information while your model is running. You may want to
+    check out the
+    <a href="https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/profile/README.md">README</a>
```

Comment:
I'd probably link to the 1.6 branch version of this file instead of the one on master, so:
https://github.com/tensorflow/tensorboard/blob/1.6/tensorboard/plugins/profile/README.md

The link to master is more likely to someday end up broken, e.g. if we for example renamed the plugin or the "plugins" directory to something else.  Linking to 1.6 should be more durable over time, so that even if users are running old versions of TensorBoard with this error message, the links will still work.

Reply:
Done!